receiver/chronyreceiver/testdata/config.yml

```diff
@@ -0,0 +1,18 @@
+---
+receivers:
+  chrony/custom:
+    endpoint: "udp://localhost:3030"
+    timeout: 10s
+
+processors:
+  nop:
+
+exporters:
+  nop:
+
+service:
+  pipelines:
+    metrics:
+      receivers: [chrony/custom]
+      processors: [nop]
+      exporters: [nop]
```

Comment:
If you are going to change this PR for some other reason, add a new line here. I know some people care about that :-)